Update import-customer-data.md (#475)
vtexdocs/dev-portal-content: docs/guides/Integration-Guides/erp-integration-guide/import-customer-data.md

When implementing an ecommerce project, it may be necessary to import customer data from other systems, such as Customer Relationship Management (CRM) applications.

> VTEX Checkout natively stores shopper data in [Master Data v1](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw). Because of this, using the [Master Data v1 API](https://developers.vtex.com/docs/api-reference/masterdata-api) is the best way of integrating your customer data base to VTEX.

Reviewer: ```suggestion
The **Checkout** module stores shopper data in [**Master Data v1**](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw) by default. Because of this, using the [**Master Data v1** API](https://developers.vtex.com/docs/api-reference/masterdata-api) is the best way to integrate your customer database into VTEX.
```

In this guide, you can learn how to import customer data to VTEX. Learn more about Master Data and its features in the [Master Data developers guide](https://developers.vtex.com/docs/guides/master-data-how-it-works).

>ℹ️ Importing your customers' data will also enable automatic fill-in at checkout for those customers. Learn more about [SmartCheckout - Customer information automatic fill-in](https://help.vtex.com/en/tutorial/smartcheckout-preenchimento-automatico-de-dados-do-cliente--2Nuu3xAFzdhIzJIldAdtan#)

## Data entities

> Master Data v1 leveradges the concept of [data entities](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw#data-entities) to allow for highly customizable data bases. These work as tables, where each document (profile or address) is saved as a row.

Reviewer: ```suggestion
**Master Data v1** leverages the concept of [data entities](https://help.vtex.com/en/tutorial/master-data--4otjBnR27u4WUIciQsmkAw#data-entities) to allow for highly customizable databases. These work as tables, where each document (profile or address) is saved as a row.
```
